autoload/codeium.vim

```diff
@@ -412,7 +412,7 @@ function! codeium#RedrawStatusLine() abort
 endfunction
 
 function! codeium#ServerLeave() abort
-  if !get(g: ,'codeium_server_job')
+  if !exists('g:codeium_server_job')
```

Comment:
```suggestion
  if !exists('g:codeium_server_job') || g:codeium_server_job is v:null
```
Does this make sense? (given this line: https://github.com/Exafunction/Exafunction/blob/ba49d6cc43d83b2484574446722a271fb991e012/exa/extensions/codeium.vim/autoload/codeium/server.vim#L28)